packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/BasicBarGraphs/AxisInfoTextLabels.jsx

```diff
@@ -25,7 +25,7 @@ export default () => {
   return (
     <React.Fragment>
       <div id="tooltip" className="initial-tooltip" />
-      <div id="axisInfoTextLabelsBargraph"> </div>
+      <div id="axisInfoTextLabelsBargraph" class="example-graph-container" />
```

Comment:
This and the others should be using `className` rather than `class`.

Reply:
Fixed here: 10a89bea8443641f496a6729a16f15e422cf90a8